package.json

```diff
@@ -106,6 +106,8 @@
     "start:server:e2e-on-build": "node server 8080",
     "start:server": "cross-env NODE_CONFIG_ENV=development nodemon server 8080",
     "debug:server": "cross-env NODE_CONFIG_ENV=development nodemon --inspect server.js 8080",
+    "profile:server": "cross-env NODE_CONFIG_ENV=development node --prof server 8080",
+    "benchmark:badge": "node scripts/benchmark-performance.js --iterations 10100 | node scripts/capture-timings.js --warmup-iterations 100",
```

Comment:
I tried running this and for me it throws

```
events.js:288
      throw er; // Unhandled 'error' event
      ^

Error: listen EACCES: permission denied 127.0.0.1:80
    at Camp.setupListenHandle [as _listen2] (net.js:1292:21)
    at listenInCluster (net.js:1357:12)
    at GetAddrInfoReqWrap.doListen [as callback] (net.js:1496:7)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:69:10)
Emitted 'error' event on WebSocketServer instance at:
    at Camp.emit (events.js:311:20)
    at Camp.EventEmitter.emit (domain.js:482:12)
    at emitErrorNT (net.js:1336:8)
    at processTicksAndRejections (internal/process/task_queues.js:84:21) {
  code: 'EACCES',
  errno: 'EACCES',
  syscall: 'listen',
  address: '127.0.0.1',
  port: 80
}
Average time over -100 iterations: 0ms
```

Before I spend a lot of time trying to track this down, can you reproduce, or is it just me?
Looking at the diff, I'm surprised we're trying to bind to port 80 here..

Reply:
Could you please try again? This was probably because `NODE_CONFIG_ENV` was missing - I've got a _local.yml_ config which probably hid that problem.

Reply:
Thanks. I still get the same issue with the latest changes, but it works if I set the port in an env var e.g: `PORT=1111 npm run benchmark:badge`. I think you need to add that to the NPM command because we're not explicitly passing a port to `node server` like we usually would.

Reply:
I've switched to `NODE_CONFIG_ENV=test` - that one contains a port and seems to be what is used for most scripts that actually need to start the server.